cpp/src/arrow/util/bit_stream_utils.h

```diff
@@ -207,9 +207,9 @@ class BitReader {
 };
```

Comment:
We could also try to make this file internal IMHO.

Reply:
Renamed this one and its transitive including `rle_encoding.h`.